src/content/blog/2020-12-07-roadmap-2021.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Roadmap 2021 (2020-12-07)
+sort: -202012070
+contributors:
+    - sokra
+---
+
+It has been nearly 2 months since webpack 5 has been officially released. Due to the sponoring situtation we couldn't spend as much time on webpack as we would like too. Speaking only for myself (@sokra) I enjoyed the little break and worked on a little side project. Ironically I used webpack 5 and all the bleeding-edge features (asset modules, worker support, persistent caching), so I discovered a few more bugs in webpack 5. So did a people during upgrading there projects to webpack 5, which led to a lot work going into bugfixing. Here is a little summary:
```

Comment:
```suggestion
It has been nearly 2 months since webpack 5 has been officially released. Due to the sponsoring situation, we couldn't spend as much time on webpack as we would like to. Speaking only for myself (@sokra) I enjoyed the little break and worked on a little side project. Ironically I used webpack 5 and all the bleeding-edge features (asset modules, worker support, persistent caching), so I discovered a few more bugs in webpack 5. So did a people during upgrading their projects to webpack 5, which led to a lot of work going into bug fixing. Here is a little summary:
```